content/documentation/staging/tutorial/03-first-steps/03-open-application-to-outside.adoc

```diff
@@ -0,0 +1,97 @@
+
+icon:bullhorn[size=1x]{nbsp} There are multiple ways to create a DNS entry depending of the platform, servers or services that you are using.
+
+For minikube we will check the External IP of the Ingress gateway:
+
+[source,bash]
+----
+$ kubectl get services/istio-ingressgateway -n istio-system
+NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)                                                                      AGE
+istio-ingressgateway   LoadBalancer   10.101.6.144   10.101.6.144   15021:30757/TCP,80:32647/TCP,443:30900/TCP,31400:30427/TCP,15443:31072/TCP   19h
+----
+
+And we will create a simple entry in the `/etc/hosts` of the tutorial machine with the desired DNS entry:
+
+[source,bash]
+----
+...
+10.101.6.144 control.travel-control.istio-cluster.org
+...
+----
+
+Then from this machine, the url `control.travel-control.istio-cluster.org` will resolve to the External IP of the Ingress Gateway of Istio.
+
+NOTE: *[2]* Use the Request Routing Wizard on *control* service to generate a traffic rule
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-service-actions.png" title="Request Routing Wizard">
+    <img src="/images/tutorial/03-03-service-actions.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Any request will be routed to the *control* workload.
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-request-routing.png" title="Routing Rule">
+    <img src="/images/tutorial/03-03-request-routing.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Use the Advanced Options and add a gateway with host `control.travel-control.istio-cluster.org` and create the Istio config.
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-create-gateway.png" title="Create Gateway">
+    <img src="/images/tutorial/03-03-create-gateway.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Verify the Istio configuration generated.
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-istio-config.png" title="Istio Config">
+    <img src="/images/tutorial/03-03-istio-config.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[3]* Test the *control* service
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-test-gateway.png" title="Test Gateway">
+    <img src="/images/tutorial/03-03-test-gateway.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[4]* Review *travel-control* namespace in Kiali
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-travel-control-graph.png" title="Travel Control Graph">
+    <img src="/images/tutorial/03-03-travel-control-graph.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Understanding what happened:
+
+. Extenal traffic enters into the cluster through a Gateway
+. Traffic is routed to *control* service through a VirtualService
```

Comment:
the *control*

Here you introduce VirtualService for the first time.  Perhaps above you should mention above what the wizard will create for you.

Reply:
I've added a reference to the VirtualService but the Wizard scenario and what generates is described in the "connect" step a couple of points later, if I introduce this here it triggers to add more context that is reserved to other section.